subworkflows/nf-core/gatk_tumor_normal_somatic_variant_calling/meta.yml

```diff
@@ -0,0 +1,127 @@
+name: gatk_tumor_normal_somatic_variant_calling
+description: |
+  Perform variant calling on a paired tumor normal set of samples using mutect2 tumor normal mode.
+  f1r2 output of mutect2 is run through learnreadorientationmodel to get the artifact priors.
+  Run the input bam files through getpileupsummarries and then calculatecontamination to get the contamination and segmentation tables.
+  Filter the mutect2 output vcf using filtermutectcalls, artifact priors and the contamination & segmentation tables for additional filtering.
+keywords:
+  - gatk4
+  - mutect2
+  - learnreadorientationmodel
+  - getpileupsummaries
+  - calculatecontamination
+  - filtermutectcalls
+  - variant_calling
+  - tumor_only
+  - filtered_vcf
+modules:
+  - gatk4/mutect2
+  - gatk4/learnreadorientationmodel
+  - gatk4/getpileupsummaries
+  - gatk4/calculatecontamination
+  - gatk4/filtermutectcalls
+input:
+  - meta:
+      type: map
+      description: |
+        Groovy Map containing sample information
+        e.g. [ id:'test' ]
+  - input:
+      type: list
+      description: list containing the tumor and normal BAM files, in that order, also able to take CRAM as an input
+      pattern: "[ *.{bam/cram} ]"
+  - input_index:
+      type: list
+      description: list containing the tumor and normal BAM file indexes, in that order, also able to take CRAM index as an input
+      pattern: "[ *.{bam.bai/cram.crai} ]"
+  - which_norm:
+      type: list
+      description: optional list of sample headers contained in the normal sample input file.
+      pattern: "testN"
+  - fasta:
+      type: file
+      description: The reference fasta file
+      pattern: "*.fasta"
+  - fastaidx:
+      type: file
+      description: Index of reference fasta file
+      pattern: "fasta.fai"
+  - dict:
+      type: file
+      description: GATK sequence dictionary
+      pattern: "*.dict"
+  - germline_resource:
+      type: file
+      description: Population vcf of germline sequencing, containing allele fractions.
+      pattern: "*.vcf.gz"
+  - germline_resource_idx:
```

Comment:
@FriederikeHanssen I think I usually go for `_tbi` for the tabix index.
What's your take on that?

Reply:
yes lets stick with tbi, think this is what we used in strelka etc as well

Reply:
Will need to change this in a few modules I think, will try remember to do that when we update the test data. Just to clarify, it should be "germline_resource_tbi"? Assuming it should be the same for "fastaidx" and "panel_of_normals_idx" as well?

Reply:
yes, `panel_of_normals_idx` -> `panel_of_normals_tbi` or even `pon_tbi` actually.
and `fastaidx` -> `fai`

Reply:
I like panel of normal. We have complete names for the other files as well